### DISCUSSION TITLE
Project documentation

### DISCUSSION THREAD

**Discussion by ErnaneJ** [Thiteago/svelte-clau-prado#6] [Q&A]
The project's Readme is out of date with what the application actually looks like today. Do you intend to update it? Or is this documentation still enough? It seems to me that it comes by default in svelte applications. I got confused when running it.

**Comment by Thiteago** [comment]
The readme will be update soon. The project are development only by myself, so there is a lot of essencial functions that i prefer to priorize for  now, but i think in one  month more or  less, it  will be properly updated,  thank you for  your consern. stay tuned for future updates
